x-pack/plugins/infra/public/pages/link_to/create_query_string_for_detail_time.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Location } from 'history';
+import { getFromFromLocation, getToFromLocation } from './query_params';
+
+export const createQueryStringForDetailTime = (location: Location) => {
+  const to = getToFromLocation(location);
+  const from = getFromFromLocation(location);
+  return to && from
+    ? `?metricTime=(autoReload:!f,time:(from:${from},interval:>%3D1m,to:${to}))`
```

Comment:
It would really be awesome if we could abstract away that manual rison construction using the `replaceStateKeyInQueryString` function, but this works too.